internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -818,6 +818,22 @@ components:
             for the OS itself (they will not be available for the build root or
             any other part of the build process). The package_sets field for these
             repositories is ignored.
+        filesystem:
+          type: array
+          items:
+            $ref: '#/components/schemas/Filesystem'
+    Filesystem:
+      type: object
+      required:
+        - mountpoint
+        - min_size
+      properties:
+        mountpoint:
+          type: string
+          example: '/var'
+        min_size:
```

Comment:
You can specify a type to use using the `x-go-type` field. So `x-go-type: uint64` will remove the need to do any conversion and any theoretical size limit is honored.

Reply:
Oh, cool! Let's do that in a follow-up?